### ISSUE TITLE
int64 backed int100

### ISSUE THREAD

**Issue by mortendahl** [mortendahl/tf-encrypted#241]
clean up interfaces between int100 and crt at the same time

**Comment by mortendahl** [comment]
yeah exactly; five residues should give the same as we have now with ten. Some of the code for recombining values needs to be revisited to make sure things work out, but it should be straight forward and may not require any code changes.

Thought this could be a nice time to finally introduce a CrtTensor type, which int100 will just be an instance of or wrap

**Comment by mortendahl** [comment]
We should still implement this but note that CrtTensors have gone somewhat out of fashion lately. They may be reintroduced for larger models again though.

**Comment by jvmancuso** [comment]
This is pretty low effort, no? The CRT tensor has been generalized from what I can tell, so it seems it's just a matter of fiddling with the moduli

**Comment by mortendahl** [comment]
Yeah it should be very straight forward; these moduli could work: 1999703, 1990007, 1996949, 1925899, 1816117
